From cebb3c9f16a29aa62bd4b44a1cd9da19841f23ef Mon Sep 17 00:00:00 2001
From: charitardha <satya.charitardha.jayanti@intel.com>
Date: Sat, 23 Feb 2013 00:05:48 +0530
Subject: [PORT FROM MAIN][DO NOT UPSTREAM][Background Music] Support streaming of music to

WIDI/ HDMI sinks simultaneosuly along with other sinks

BZ: 63786

A new key for background music support over WIDI/HDMI is defined
in audio.h to enable/disable this BGM based on settings
set in the UI.

Change-Id: I1c0e09c1ea0a18de13029ccc2e96613d7d8a6e75
Orig-Change-Id: Ic0c33107c13b1c95de93a3db40414f2a319b1c6b
Signed-off-by: charitardha <satya.charitardha.jayanti@intel.com>
Reviewed-on: http://android.intel.com:8080/93394
Reviewed-by: cactus <cactus@intel.com>
Reviewed-by: Krishna, Samaga <samaga.krishna@intel.com>
Reviewed-by: Sikkandar D, Madar <madar.sikkandar.d@intel.com>
Reviewed-by: Anand, Jerome <jerome.anand@intel.com>
Tested-by: Gupta, ArvindX K <arvindx.k.gupta@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 include/hardware/audio.h |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/include/hardware/audio.h b/include/hardware/audio.h
index 66b592c..8e16e4e 100644
--- a/include/hardware/audio.h
+++ b/include/hardware/audio.h
@@ -110,6 +110,9 @@ __BEGIN_DECLS
 /* Stream Flags */
 #define AUDIO_PARAMETER_KEY_STREAM_FLAGS "stream_flags"
 
+/* Remote BGM state */
+#define AUDIO_PARAMETER_KEY_REMOTE_BGM_STATE "bgm_state"
+
 /**
  *  audio stream parameters
  */
-- 
1.7.4.1

